test/test_attention.py

```diff
@@ -172,7 +172,7 @@ def test_get_context_and_attention_probs():
 
     # data
     source_nd = mx.nd.random_normal(shape=(batch_size, seq_len, num_hidden))
-    source_length_np = np.random.random_integers(1, seq_len, (batch_size,))
```

Comment:
[numpy.random.random_integers](https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.random_integers.html) is deprecated